src/connector/sqlite.rs

```diff
@@ -10,18 +10,18 @@ use crate::{
     visitor::{self, Visitor},
 };
 use async_trait::async_trait;
-use rusqlite::NO_PARAMS;
-use std::{collections::HashSet, convert::TryFrom, path::Path, time::Duration};
+use std::{convert::TryFrom, path::Path, time::Duration};
 use tokio::sync::Mutex;
 
-pub(crate) const DEFAULT_SQLITE_SCHEMA_NAME: &str = "quaint";
+pub(crate) const DEFAULT_SQLITE_SCHEMA_NAME: &str = "main";
 
 /// A connector interface for the SQLite database
 #[cfg_attr(feature = "docs", doc(cfg(feature = "sqlite")))]
 pub struct Sqlite {
     pub(crate) client: Mutex<rusqlite::Connection>,
     /// This is not a `PathBuf` because we need to `ATTACH` the database to the path, and this can
     /// only be done with UTF-8 paths. This is `None` for purely in-memory databases.
+    #[allow(dead_code)]
     pub(crate) file_path: Option<String>,
```

Comment:
Not sure if it's useful to keep that field around as it's a public struct that could be used in existing projects

Reply:
`pub(crate)` means it's not exposed outside the crate, so if you get a warning it's not used anywhere, for sure.

Reply:
And you should not expose fields from a public crate. If you want to give an access to the data, you'd use a method instead, allowing us to change their way of working without breaking any public apis.

Reply:
Good, I'll get rid of it before merging